docs/index.md

```diff
@@ -2524,7 +2522,8 @@ cotangent. So let's retrieve their definitions.
 Let's first find the definition of the tangent.
 
 > Select the icon <img width="16" height="16" src="search.svg" alt="search"> (search).
-> In the pattern enter <tt>( tan &#96;</tt> and click on Search.
+> In the pattern enter <tt>( tan &#96;</tt>, change type to <tt>|-</tt> if needed, 
```

Comment:
In my case `class` was selected by default in the search dialog, so I decided to mention this.